pyClient/test_commands/test_mimc_hash.py

```diff
@@ -15,10 +15,10 @@ def test_mimc_round() -> None:
     print("Test Round passed")
 
 
-def test_sha3() -> None:
-    assert sha3_256("Clearmatics") == \
+def test_keccak256() -> None:
```

Comment:
This test was here before. I think it is to test that the encoding is correct. It might also have been there to check the value of the first round constant in MiMC long ago (we use different seeds now).

Reply:
How has the value `14220067918847996031108144435763672811050758065945364308986253046354060608451` been generated?

Reply:
I checked with Giuseppe and we should have generated it with pysha3 package.

Reply:
This can help too: https://github.com/rrtoledo/mimc/blob/master/mimc.py#L14

Reply:
I see, thanks guys.
Not sure the file https://github.com/clearmatics/zeth/blob/develop/pyClient/zeth/mimc.py is useful. If not, let's remove it ("less is more"), and thus we can remove the associated test.

Reply:
Removed